libs/category/src/drivers/magento/transformers/applied-filter-transformer.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+
+import { MagentoCategoryFilters } from '../models/requests/filters';
+import { DaffCategoryFilterAction } from '../../../models/requests/filter-action';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffMagentoAppliedFiltersTransformService {
+
+  transform(daffFilters: DaffCategoryFilterAction[]): MagentoCategoryFilters {
+		const magentoFilters: MagentoCategoryFilters = {};
+
+		daffFilters.map(filter => {
```

Comment:
I think you should use `forEach` instead of `map` if you're not using the mapped array.

Reply:
changed